lib/src/config/acl.rs

```diff
@@ -0,0 +1,67 @@
+use crate::acl;
+
+#[derive(Clone, Debug, Default)]
+pub struct AclConfig(pub(crate) acl::Acls);
+
+mod deserialization {
```

Comment:
It seems fine, but I'm curious: is there a reason to put this into its own `deserialization` module? And would it be worth putting it into its own `lib/src/config/acl/deserialization.rs`?

Reply:
I choose this structure to match how other similar sections (e.g [`backends`](https://github.com/fastly/Viceroy/blob/6aae133d0a75cf423064b07c40ccce8830ee7e98/lib/src/config/backends.rs#L31), [`device_detection`](https://github.com/fastly/Viceroy/blob/6aae133d0a75cf423064b07c40ccce8830ee7e98/lib/src/config/device_detection.rs#L38), [`dictionaries`](https://github.com/fastly/Viceroy/blob/6aae133d0a75cf423064b07c40ccce8830ee7e98/lib/src/config/dictionaries.rs#L99), ...). So not a very compelling reason.

I'll change it to whichever organization you think best.